app.js

```diff
@@ -41,6 +41,12 @@ app.get('/crit/:crit_id', (req, res) => {
 	});
 	
 });
+//welcome page route

```

Comment:
Add an extra line above this. Give a little bit of breathing room between routes.